components/LinkAdd.tsx

```diff
@@ -0,0 +1,42 @@
+import styles from "@/styles/LinkAdd.module.css";
+import link from "@/public/link.svg";
+import { useState } from "react";
+import Image from "next/image";
+
+interface ToggleModal {
+  toggleModal: (id: number) => void;
+}
+
+function LinkAdd({ toggleModal }: ToggleModal) {
+  const [inputValue, setInputInput] = useState("");
+
+  function handleInputChange(event: React.ChangeEvent<HTMLInputElement>) {
```

Comment:
`React.`은 없이 표현해주는게 더 좋을 것 같아요!
```suggestion
  function handleInputChange(event: ChangeEvent<HTMLInputElement>) {
```